crates/storage/Cargo.toml

```diff
@@ -10,13 +10,14 @@ rust-version = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 base64 = { workspace = true }
-bincode = "2.0.0-rc.3"
+bincode = { version = "2.0.0-rc.3", features = ["serde"] }
 bitvec = { workspace = true }
 bloomfilter = "1.0.12"
 cached = { workspace = true }
 const_format = { workspace = true }
 data-encoding = "2.4.0"
 fake = { workspace = true }
+flume = "0.11.0"
```

Comment:
Nothing against `flume` itself - is there a specific reason you want this over the std channels? Including this would mean we have tokio, std and flume for channels - and I think `flume` itself depends on tokio.

Reply:
Ah I missed that! Apologies.